cli/dts/lib.deno.ns.d.ts

```diff
@@ -3010,4 +3013,94 @@ declare namespace Deno {
     recordType: RecordType,
     options?: ResolveDnsOptions,
   ): Promise<string[] | MXRecord[] | SRVRecord[] | string[][]>;
+
+  export interface CommandOptions {
+    /** Arguments to pass to the process. */
+    args?: string[];
+    /**
+     * The working directory of the process.
+     * If not specified, the cwd of the parent process is used.
+     */
+    cwd?: string | URL;
+    /**
+     * Clear environmental variables from parent process.
+     * Doesn't guarantee that only `opt.env` variables are present,
+     * as the OS may set environmental variables for processes.
+     */
+    clearEnv?: boolean;
+    /** Environmental variables to pass to the subprocess. */
+    env?: Record<string, string>;
+    /**
+     * Sets the child process’s user ID. This translates to a setuid call
+     * in the child process. Failure in the setuid call will cause the spawn to fail.
+     */
+    uid?: number;
+    /** Similar to `uid`, but sets the group ID of the child process. */
+    gid?: number;
+
+    stdin?: "piped" | "inherit" | "null";
+    stdout?: "piped" | "inherit" | "null";
+    stderr?: "piped" | "inherit" | "null";
+  }
+
+  /**
+   * Spawns a child process.
+   *
+   * stdin defaults to null.
+   * stdout defaults to inherit.
+   * stderr defaults to inherit.
+   */
+  export function spawn<T extends CommandOptions = CommandOptions>(
+    command: string | URL,
+    options?: T,
+  ): Child<T>;
+
+  export class Child<T extends CommandOptions> {
+    readonly stdin: T["stdin"] extends "inherit" | "null" ? null
+      : WritableStream<Uint8Array>;
+    readonly stdout: T["stdout"] extends "inherit" | "null" ? null
+      : ReadableStream<Uint8Array>;
+    readonly stderr: T["stderr"] extends "inherit" | "null" ? null
+      : ReadableStream<Uint8Array>;
+
+    readonly pid: number;
+    /** Get the current status. */
+    readonly status: ProcessStatus | null;
+
+    /** Waits for the child to exit completely, returning the status that it exited with. */
+    wait(): Promise<ProcessStatus>;
+    /** Waits for the child to exit completely, returning all its output and status. */
+    output(): Promise<CommandOutput<T>>;
+    kill(signo: Signal): void;
```

Comment:
Is `signo` a required argument? If not, please add a docstring saying which signal is used by default

Reply:
Nope, its not optional, just like with `Deno.run`. Do you want it to be optional?

Reply:
Nope. Makes sense then. please add a docstring